frontend/src/components/MainToolbar.vue

```diff
@@ -236,4 +272,14 @@ export default {
     color: white;
     font-weight: bold;
   }
+
+  .extension-expansion-panel {
+    max-height: 20vh;
```

Comment:
Why unit vh?

Reply:
I chose ch to expand the main menu height dynamically, depending on the browser size. Anyway, this will be removed when we move the extension information to another dialog

Reply:
removed completely as we no longer show the list in this PR